src/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+}
+
+body > * {
+  width: 300px;
+  height: 100px;
+  margin: auto;
+}
```

Comment:
do not increase selector specificity without extra need;
use specific selectors, not `*`